src/Configuration.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection;
+
+use PhpParser\Lexer\Emulative;
+use PhpParser\Parser;
+use PhpParser\ParserFactory;
+use Roave\BetterReflection\NodeCompiler\CompileNodeToValue;
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\Reflector\FunctionReflector;
+use Roave\BetterReflection\SourceLocator\Ast\Locator as AstLocator;
+use Roave\BetterReflection\SourceLocator\Ast\Strategy\AstConversionStrategy;
+use Roave\BetterReflection\SourceLocator\Ast\Strategy\NodeToReflection;
+use Roave\BetterReflection\SourceLocator\Type\AggregateSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\AutoloadSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\EvaledCodeSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\PhpInternalSourceLocator;
+use Roave\BetterReflection\SourceLocator\Type\SourceLocator;
+use Roave\BetterReflection\Util\FindReflectionOnLine;
+
+// @TODO should the naming be swapped around? `new BetterReflection()` sounds MUCH cooler!
```

Comment:
Currently, the logic is `new \Roave\BetterReflection\Configuration()`. In my opinion, `new \Roave\BetterReflection()` would be quite awesome

Reply:
Yes, that would be nice :)